content/angular/step11.md

```diff
@@ -12,6 +12,9 @@ To do so, we'll add a [test driver](http://guide.meteor.com/testing.html#test-dr
 meteor add practicalmeteor:mocha
 ```
 
+> ### New testing feature
+> While the `meteor test ...` command does still work as specified below, the release of Meteor 1.7 includes a new feature which allows you to specify the location of the test module in `package.json` with a property named `"testModule"`, which you can read more about [in the changelog](https://docs.meteor.com/changelog.html#v1720180528). In order to get the expected behavior, such that the `meteor test ...` command only uses files whose names match the format `*.test[s].*` or `*.spec[s].*`, you should either remove the line `"testModule": "tests/main.js"` from your `package.json` file, or change it to an appropriate value, before running `meteor test ...`.
```

Comment:
Should we mention that `"testModule"` need to appear inside a `"meteor"` section of `package.json`?

Reply:
That's not a bad idea. I considered it, but the section was already pretty long, so I decided against. I could go either way on it, though.